src/unleash-proxy.ts

```diff
@@ -239,7 +242,11 @@ export default class UnleashProxy {
     getAllToggles(req: Request, res: Response<FeaturesSchema | string>): void {
         const apiToken = req.header(this.clientKeysHeaderName);
 
-        if (!this.ready) {
+        if (!this.enableAllEndpoint) {
+            res.status(501).send(
+                '/all endpoint is disabled. Please, check the configuration options of the server.',
```

Comment:
Same comment regarding the URL here as before: `/all` or `/proxy/all`? Or something else? 

But also, can we make it more helpful to the end user?

```suggestion
                'The /proxy/all endpoint is disabled. Please check your server configuration. To enable it, set the `enableAllEndpoint` configuration option or `ENABLE_ALL_ENDPOINT` environment variable to `true`.'
```